lib/chef/resource/cron_d.rb

```diff
@@ -162,7 +163,7 @@ def self.validate_dow(spec)
 
       # warn if someone passes the deprecated cookbook property
       def after_created
-        raise ArgumentError, "The 'cookbook' property for the cron_d resource is no longer supported now that this resource ships in Chef itself." if cookbook
+        raise ArgumentError, "The 'cookbook' property for the cron_d resource is no longer supported now that this resource ships in #{Chef::Dist::PRODUCT} itself." if cookbook
```

Comment:
Maybe another place we can omit the reference?

```suggestion
        raise ArgumentError, "The 'cookbook' property for the cron_d resource is no longer supported now that cron_d ships as a core resource." if cookbook
```